src/ts_catalog/catalog.h

```diff
@@ -1484,6 +1486,10 @@ typedef struct FormData_job_error
 
 typedef FormData_job_error *Form_job_error;
 
+#define TIERING_STATUS_DEFAULT 0
+#define TIERING_STATUS_ENABLED 1
+#define TIERING_ALLOW_ORDERED_APPEND 2
```

Comment:
can we make this a bit more generic and tiering being just 1 set of flags similar to CHUNK_STATUS

Reply:
i would make the flag not about ordered append but about overlapping chunks

Reply:
done